docs/install.rst

```diff
@@ -60,9 +60,9 @@ Installing Astropy
 Using pip
 ---------
 
-To install Astropy with `pip <http://www.pip-installer.org/en/latest/>`_, simply run::
+To install Astropy with `pip <https://pip.pypa.io>`__, simply run::
```

Comment:
Why two trailing underscores?

Reply:
The link is used lots of times in this `rst`. Would it make sense to define it just once and reuse it?

Reply:
There were warnings related to this link and the only way I could fix it was by making all the links 'anonymous' with the double underscore. This appeared to be related to other definitions of the pip link elsewhere in the docs. I don't have time to investigate, but I propose that if anyone cares enough, they can open a follow-up PR?

Reply:
Ah, I didn't knew about anonymous links. Please ignore my previous comments 👍